rasa/nlu/test.py

```diff
@@ -398,6 +398,41 @@ def evaluate_response_selections(
     }
 
 
+def _add_confused_intents_to_report(
+    report: Dict[Text, Dict[Text, float]],
+    cnf_matrix: np.ndarray,
+    labels: Iterable[Text],
```

Comment:
it's iterable + `Sized` (since you can apply `len(...)` to it 
```suggestion
    labels: Collection[Text],
```